packages/corelib/src/worker/studio.ts

```diff
@@ -36,10 +37,16 @@ export enum StudioJobs {
 	RemovePlaylist = 'removePlaylist',
 	RegeneratePlaylist = 'regeneratePlaylist',
 
+	/** @deprecated use OnPlayoutPlaybackChanged instead */
```

Comment:
Why are we keeping these deprecated methods?
We already require server-core-integration to be the in the same semver minor range.

Im worried that there is some complexity added by keeping these methods around, and if we don't clean it up now then I dont know when we will

Reply:
If we're going to take these Core and Playout Gateway changes simultaneously, would these methods even be reachable? I don't think we have any other devices that could call them.

Reply:
Ok, I will remove them